src/App.tsx

```diff
@@ -1,62 +1,117 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
+
+enum SortType {
+  Dumplings = 'Dumplings',
+  Carrot = 'Carrot',
+  Eggs = 'Eggs',
+  IceCream = 'Ice cream',
+  Apple = 'Apple',
+  Bread = 'Bread',
+  Fish = 'Fish',
+  Honey = 'Honey',
+  Jam = 'Jam',
+  Garlic = 'Garlic',
+}
+
+function getPreparedGoods(
+  goods: SortType[],
+  sortField: string,
+  isReversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_FIELD_ALPHABET:
```

Comment:
```suggestion
        case enumName.keyName
```
do it in second case too